scripts/performance/perf_load/perf_req_gen_config_ledger.py

```diff
@@ -0,0 +1,17 @@
+import libnacl
+from indy import ledger
+
+from perf_load.perf_utils import rawToFriendly
+from perf_load.perf_req_gen import RequestGenerator
+
+
+class RGConfig(RequestGenerator):
+    _req_types = ["111"]
+
+    def _rand_data(self):
```

Comment:
this functions generates random data for the req. For config type it is enough to generate just random int like this
def _rand_data(self):
      return str(random.randint(0, 99999999))

Reply:
Fixed.